backend/app/api/v1/endpoints/cpt/cptJobs.py

```diff
@@ -169,6 +169,8 @@ async def filters(
 
     updated_filter_qs = urlencode(filter_dict, doseq=True) if filter else ""
 
+    updated_filter_qs = urlencode(filter_dict, doseq=True) if filter else ""
```

Comment:
Why is this line duplicated before the debug `print`? (And of course we probably don't want to merge with the `print` anyway.)